app/models/grda_warehouse/hud/client.rb

```diff
@@ -1033,25 +1037,26 @@ def show_health_hpc_for?(user)
     # and maintained in the warehouse
     def self.full_release_string
       # Return the untranslated string, but force the translator to see it
-      if GrdaWarehouse::Config.implicit_roi?
-        Translation.translate('Implicit Release')
-        'Implicit Release'
-      else
-        Translation.translate('Full HAN Release')
-        'Full HAN Release'
-      end
+      Translation.translate(active_consent_model.full_release_string)
+      active_consent_model.full_release_string
     end
 
     def self.partial_release_string
       # Return the untranslated string, but force the translator to see it
-      Translation.translate('Limited CAS Release')
-      'Limited CAS Release'
+      Translation.translate(active_consent_model.partial_release_string)
+      active_consent_model.partial_release_string
```

Comment:
To make this work, I think we want class methods for `partial_release_string` on the consent models.  The instance methods can just return the class method value.

```
def self.partial_release_string
  'the string'
end

def partial_release_string
  self.class.partial_release_string
end
```

That way we can just call 
```
GrdaWarehouse::Config.active_consent_class.partial_release_string
```